docs/installation.md

```diff
@@ -35,7 +35,10 @@ title: Installation
 
 1. `npm install react-native-camera --save`
 2. `react-native link react-native-camera`
+3. React Native RN >= 0.60.X: Run `pod install` in the `ios` directory.
    _To install it with Windows, see manual install below_
+   
+For React Native RN >= 0.60.X you will be warned about `react-native-camera` and auotlinking. It is safe to ignore this error.
```

Comment:
Since RN 0.60, modules that have podspec (like this one), don't ever need running `react-native link`. What's confusing is step 2 – it should be removed for RN 0.60+ and only shown for older versions. 

RN CLI warns you about manually linked deps, because they are needless and may cause duplicates being found by Xcode, preventing you from build.

Reply:
@browniefed you can handle the changes proposed by @thymikee ?